File: C Program/Day 3/pw/loops/prime.c
#include <stdio.h>
int main()
{
    int p;
    int num;
    int a = 0;
    printf("Enter they number \n");
    scanf("%d", &num);
    if (num == 1)
    {
        printf("Number is Nieter prime or composite\n");
    }

    for (int i = 2; i < num; i++)
    {
        if (num % i == 0)
        {
            
            a = 1;
            break;
        }
    }
    if (a == 0)
    {
        printf("prime number %d\n", num);
    }
    else
    {
        printf("Composite number %d\n", num);
    }

    return 0;
}

File: JAVA/Practice/Prime.java
import java.util.*;
class Prime{
	public static void main(String[] args) {
		System.out.println("Enter the number");
		Scanner sc = new Scanner(System.in);
		int num = sc.nextInt();

		for (int i=2;i<9;i++ ) {
			if (num % i == 0) {
				System.out.println(+num +"Not prime number");
				return;
			}
			else{
				System.out.println(+num +"It is prime number");
				return;
			}
		}
	}
}